include/tvm/runtime/packed_func.h

```diff
@@ -223,6 +223,10 @@ class ExtTypeVTable {
 class TVMPODValue_ {
  public:
   operator double() const {
+    // Allow automatic conversion from int to float
+    if (type_code_ == kDLInt) {
```

Comment:
Is this to make the attributes more flexible?

Reply:
Yes, for example, user can do add_one(x, 1), and in the backend we can directly fetch out a float, for functions that want to do so